streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -439,7 +439,7 @@
                     RECONNECT_BACKOFF_MS_DOC)
             .define(RECONNECT_BACKOFF_MAX_MS_CONFIG,
                     ConfigDef.Type.LONG,
-                    50L,
+                    1000L,
```

Comment:
When Dana implemented exponential backoff, she disabled it for Streams and Connect. I don't think there's a good reason not to enable it for those two and hence this PR. In the Streams case, I think those two configs are actually used for the `StreamsKafkaClient`.